core_lib/interface/timecontrols.cpp

```diff
@@ -162,6 +162,9 @@ void TimeControls::setCore( Editor* editor )
 {
     Q_ASSERT( editor != nullptr );
     mEditor = editor;
+
+    connect( mLoopStartSpinBox, spinBoxValueChanged, mEditor->playback(), &PlaybackManager::setRangedStartFrame );
+    connect( mLoopEndSpinBox, spinBoxValueChanged, mEditor->playback(), &PlaybackManager::setRangedEndFrame );
```

Comment:
These 2 connections are actually duplicated. But I understand the current signal connections of Timeline is a mess. I think the new connections are more straightforward than the olds so I'll keep this and remove the existing one.